src/store/modules/images.js

```diff
@@ -197,6 +197,15 @@ const actions = {
       else {
         recent_images[old_image_index] = new_image
       }
+      // Reassigning value of current_image to new_image
+      // If a user takes smart stack photos and they select the image as it's updating,
+      // then the selected image (i.e. the thumbnail with the surrounding yellow border) keeps the yellow border
+      let current_image = state.current_image
+      const current_image_SMARTSTK = current_image.header && current_image.header.SMARTSTK
+      const new_image_SMARTSTK = new_image.header && new_image.header.SMARTSTK
+      if (current_image_SMARTSTK && current_image_SMARTSTK !== 'no' && current_image_SMARTSTK === new_image_SMARTSTK) {
+        current_image = new_image
```

Comment:
Modifying state in vuex needs to happen by calling a mutation, so direct modifications like line 207 won't work like we want. However, replacing it with `commit('setCurrentImage', new_image)` should do the trick!

(I was trying out these changes with the [send_test_image](https://github.com/LCOGT/photonranch-helper-scripts) script I mentioned that uploads simulated smart stacks to the tst site. As far as I can tell, everything should work after making this change).

Reply:
Update--decided to push this fix directly since it's such a minor change and might make observing easier for Wayne over this weekend. Verified it works with the send_test_image script as well as live observations at aro1, so likely no further action needed.